imports/plugins/custom/gdr-landing-page/server/methods/getTrendingProducts.js

```diff
@@ -0,0 +1,28 @@
+import { Meteor } from "meteor/meteor";
+import { check } from "meteor/check";
+import { Orders } from "lib/collections";
+
+Meteor.methods({
+  /**
+   * Meteor method to get trending products in the database
+  * @param  {Number} limit - maximum amount of products to get
+  */
+  "product/getTrendingProducts": function (limit = 12) {
+    check(limit, Number);
+    const numLimit = parseInt(limit, 10) || 12;
```

Comment:
Since you passed a default parameter for limit making it 12 if no parameter is passed, i don't think this conditional check here is necessary as it may be redundant. You should use one of them but not both at the same time. You could also use `check(limit, Match.Integer)` instead of `check(limit, Number)` and then `parseInt(limit, 10)`. This is just so you know but it's very fine.

Reply:
Fixed!